david-dam/wk01/3-wed/array-function/js/script.js

```diff
@@ -0,0 +1,113 @@
+// # Array and Functions Bonus Material
+
+// 1. Define a function `maxOfTwoNumbers` that takes two numbers as arguments and returns the largest of them. 
+//    Use the if-then-else construct available in Javascript. You'll have to remember your pre-work, or do some googling to figure this out.
+
+const maxOfTwoNumbers = function(num,num2) {
+    if (num > num2) {
+        console.log(num)
+    } else {
+        console.log(num2)
+    }
+}
+
+maxOfTwoNumbers(1,2);
+maxOfTwoNumbers(5,10);
+maxOfTwoNumbers(50,100);
+
+// // 2. Define a function `maxOfThree` that takes three numbers as arguments and returns the largest of them.
+
+const maxOfThree = function(num,num2,num3) {
+    if (num > num2 && num > num3){
+        console.log(num);
+    } else if (num2 > num && num2 > num3){
+        console.log(num2);
+    } else {
+        console.log(num3);
+    }
+}
+
+maxOfThree(3,2,1);
+maxOfThree(4,6,2);
+maxOfThree(3,6,9);
+
+// 3. Write a function that takes a character (i.e. a string of length 1) and returns true if it is a vowel, false otherwise.
+
+const vowels = ['a','e','i','o','u'];
+
+    const character = function(word) {
+            if (word.length[1] = vowels) {
+                console.log(true)
+            } else {
+                console.log(false)
+            }
+    }
+
+// Can not get this to work as it only ouput as "True"
+
+    character('a')
+    character('b')
+
+// 4. Define a function `sumArray` and a function `multiplyArray` that sums and multiplies (respectively) all the numbers in an array of numbers. 
+// For example, `sumArray([1,2,3,4])` should return 10, and `multiplyArray([1,2,3,4])` should return 24.
+
+const sumArray = function(a,b,c,d) {
+    let sum = a + b + c + d;
+        console.log(sum)
+}
```

Comment:
These array functions are supposed to accept one argument, an array then use a loop to generate a result.